contracts/LidoBridgeSwapper.sol

```diff
@@ -9,14 +9,14 @@ import "./interfaces/ICurvePool.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
 /**
-* Exchanges between ETH and stETH
+* Exchanges between ETH and wStETH
 * index 0: ETH
-* index 1: stETH
+* index 1: wStETH
 */
 contract LidoBridgeSwapper is ZkSyncBridgeSwapper {
 
     // The address of the stEth token
-    address public immutable stEth;
+    address public stEth;
```

Comment:
Why remove the `ìmmutable` modifier? I suppose you do that so you can use it in the `require` on L38, but you can have both (and save a lot of gas) if you do:
```
address public immutable stEth;

constructor(....) {
        wStEth = _wStEth;
        address _stEth = IWstETH(_wStEth).stETH();
        require(_stEth == ICurvePool(_stEthPool).coins(1), "stEth mismatch");
        stEth = _stEth;
        stEthPool = _stEthPool;
        lidoReferral =_lidoReferral;
}